src/index.css

```diff
@@ -1,13 +1,58 @@
+:root {
+  --red-5: #960035;
+  --red-4: #c50035;
+  --red-3: #d71818;
+  --red-main: #ff002a;
+  --red-1: #f66060;
+  --low-prio: #707070;
+  --grey-dark-back: #212121;
+  --darker-back: #121212;
+  --grey-unselect: #c4c4c4;
+  --unselect-text: #707070;
```

Comment:
For the colors here, can we use the tailwind format

100-900 describing the different shades, where 100 is the lightest color, 900 is the darkest color for this shade

So:

`red-100`
`red-200`
`red-300`
`red-400`
`red-500`

etc.

Reply:
In fact, why don't we just use Tailwind?

Is there a reason not to use that, and match our colors to the existing colors?